### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by AnkitKashyap0709** [microsoft/vscode-java-pack#967]
# How do I locate the JDK in VS code?

**Comment by nickzhums** [comment]
could you paste where you downloaded the Java installer? and also could you please share your environment information (such as operating system, etc?)

**Comment by nickzhums** [comment]
what's the "usual downloads folder" ? which installer did you use? Did you use the coding pack for Java?

**Comment by AnkitKashyap0709** [comment]
Do u think I might have missed any step in the proper installation of it?

**Comment by nickzhums** [comment]
Thanks for the information. I'm letting @eskibear to help diagnose

**Comment by AnkitKashyap0709** [comment]
## Thanks for your effort.
### BTW, This is the configuration window inside VS Code where it prompted to configure the path.

![5](https://user-images.githubusercontent.com/63182579/162755190-9c143f11-f991-4a3e-a665-2758eb37ce60.PNG)

**Comment by nickzhums** [comment]
Did you check your "Program Files" folder, that's usually where JDK lives. for example: C:\Program Files\AdoptOpenJDK\jdk-11.0.11.9-hotspot

**Comment by AnkitKashyap0709** [comment]
The folder "AdoptOpen JDK" doesn't exist in the "Program files". That's the problem.

**Comment by AnkitKashyap0709** [comment]
Yes, I went through the ORACLE website,  and yet there was nothing helpful.
Thanks for your time and effort.

**Comment by AnkitKashyap0709** [comment]
Just wanted to know if you work at Microsoft or doing an internship there?

**Comment by Eskibear** [comment]
The default location of Eclipse Temurin should be `C:\Users\<username>\AppData\Local\Programs`.

https://github.com/microsoft/vscode-java-pack/issues/967#issuecomment-1095089526 tells that in your settings, `java.home` is pointing to an invalid location, and that's the root cause. You can remove it from your settings file and reload window, which should fix your problem.

I'm interested in it because it's a deprecated setting, did you add that setting by your own?

**Comment by AnkitKashyap0709** [comment]
Here this thing started to appear on the terminal when I run a java code.


![1](https://user-images.githubusercontent.com/63182579/162942572-d4fd6451-f029-4da0-a880-a9936cef9f33.PNG)

**Comment by Eskibear** [comment]
> Here this thing started to appear on the terminal when I run a java code.

The way you "run" your Java code is not provided by Java Extension Pack. Probably you were running your code with "Code Runner" extension. Please follow below docs:
https://code.visualstudio.com/docs/java/java-debugging

**Comment by Eskibear** [comment]
Closing as answered. The invalid java.home setting was resolved, and the remaining terminal behavior belongs to Code Runner.
